servers/navigation_server_3d.h

```diff
@@ -263,6 +263,20 @@ class NavigationServer3D : public Object {
 	NavigationServer3D();
 	virtual ~NavigationServer3D();
 
+	enum ProcessInfo {
+		INFO_ACTIVE_MAPS,
+		INFO_REGION_COUNT,
+		INFO_AGENT_COUNT,
+		INFO_LINK_COUNT,
+		INFO_POLYGON_COUNT,
+		INFO_EDGE_COUNT,
+		INFO_EDGE_MERGE_COUNT,
+		INFO_EDGE_CONNECTION_COUNT,
+		INFO_EDGE_FREE_COUNT,
+	};
+
+	virtual int get_process_info(ProcessInfo p_info) = 0;
```

Comment:
Why is this named `get_process_info`?

Reply:
It is a monitor copy from PhysicsServer, it is just the name that is used there for both functions and struc/class